transifex/utils.py

```diff
@@ -321,7 +322,7 @@ def cleanup(self):
 
         If applicable, also deletes the merged branch from GitHub.
         """
-        if self.pr and self.pr.is_merged():
+        if self.pr:
             logger.info('Deleting merged branch %s:%s.', self.name, self.branch_name)
```

Comment:
This log message should be updated, as the branch might not have been merged.

Reply:
Do you need to do anything to close the PR first if it's still open before you delete the branch?

Reply:
ok. and no deleting branch automatically closes the pull request.